src/index.css

```diff
@@ -0,0 +1,32 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Puritan:ital,wght@0,400;0,700;1,400;1,700&display=swap');
+/* stylelint-disable-next-line scss/at-rule-no-unknown */
+@tailwind base;
+/* stylelint-disable-next-line scss/at-rule-no-unknown */
+@tailwind components;
+/* stylelint-disable-next-line scss/at-rule-no-unknown */
```

Comment:
- [ ] Kindly remove every stylelint disable rule and instead fix the issue in the code.